tests/test_batch.py

```diff
@@ -183,48 +224,52 @@ def custom_get_batch_mock(request, _):
 
         # Reset history so that we can count calls later
         mock_request.reset_mock()
+        mock_request.register_uri(
+            "POST",
+            f"/core-fast/api/v1/batches/{batch.id}/jobs",
+            json=load_mock_batch_json_response,
```

Comment:
This should use the POST `/core-fast/api/v1/batches/00000000-0000-0000-0000-000000000002/jobs` fixture instead of the POST /batches fixture. 

I believe this is the reason why the assertion `mock_request.request_history[0].url == f"{self.sdk._client.endpoints.core}/api/v1/batches/{batch.id}/jobs"` stopped working

Reply:
I don't understand this comment. Do you mean this fixture?  `json=load_mock_batch_json_response`

Reply:
THis looks OK to me now. idk if it was changed or what I had in mind, resolving.